### PR TITLE
Move Theme to shared

### PR DESCRIPTION
**Pull request by MarekMacko**

### CODE REVIEW

File: androidApp/src/main/kotlin/com/intive/picover/main/theme/Theme.kt [hunk 1]
```diff
@@ -1,60 +1,29 @@
...
-
-private val LightColorScheme = lightColorScheme(
-	primary = Purple40,
-	secondary = PurpleGrey40,
-	tertiary = Pink40,
-)
+import com.intive.picover.shared.main.theme.platformThemeSideEffect
 
 @Composable
 fun PicoverTheme(
```
Comment (MarekMacko) [inline androidApp/src/main/kotlin/com/intive/picover/main/theme/Theme.kt:18]: File to be moved in a separate PR to not break git history

File: androidApp/src/main/kotlin/com/intive/picover/main/theme/Theme.kt [hunk 2]
```diff
@@ -1,60 +1,29 @@
...
-	if (!view.isInEditMode) {
-		SideEffect {
-			with(view.context as Activity) {
-				window.navigationBarColor = colorScheme.primary.copy(alpha = 0.08f).compositeOver(colorScheme.surface.copy()).toArgb()
-				window.statusBarColor = colorScheme.background.toArgb()
-				WindowCompat.getInsetsController(window, view).isAppearanceLightStatusBars = !darkTheme
-				WindowCompat.getInsetsController(window, view).isAppearanceLightNavigationBars = !darkTheme
-			}
-		}
+	val colorScheme = when (darkTheme) {
```
Comment (gmatyszczak) [inline androidApp/src/main/kotlin/com/intive/picover/main/theme/Theme.kt]: `if .. else ..`? 😄

Comment (MarekMacko) [reply androidApp/src/main/kotlin/com/intive/picover/main/theme/Theme.kt]: Fixed 😄

**Review by gmatyszczak — APPROVED** [review]
